RecoVertex/BeamSpotProducer/test/test_scalars.py

```diff
@@ -70,20 +17,30 @@
 process.maxEvents = cms.untracked.PSet(
         input = cms.untracked.int32(-1)
 )        
-
 process.load("CondCore.DBCommon.CondDBSetup_cfi")
+
+from Configuration.AlCa.GlobalTag import GlobalTag as customiseGlobalTag
+process.GlobalTag = customiseGlobalTag(globaltag = "auto:run3_hlt_GRun")
  
 process.BeamSpotDBSource = cms.ESSource("PoolDBESSource",
                                         process.CondDBSetup,
-                                        toGet = cms.VPSet(cms.PSet(
-    record = cms.string('BeamSpotObjectsRcd'),
-    tag = cms.string('BeamSpotObjects_2009_v11_offline'))),
-                                        #connect = cms.string('sqlite_file:EarlyCollision.db')
-connect = cms.string('frontier://FrontierProd/CMS_COND_31X_BEAMSPOT')
-)
-process.es_prefer_beamspot = cms.ESPrefer("PoolDBESSource","BeamSpotDBSource")
+                                        toGet = cms.VPSet(
+                                            cms.PSet(
+                                                record = cms.string('BeamSpotOnlineLegacyObjectsRcd'),
+                                                tag = cms.string("BeamSpotOnlineTestLegacy"),
+                                                refreshTime = cms.uint64(1)
+
+                                            ),
+                                            cms.PSet(
+                                                record = cms.string('BeamSpotOnlineHLTObjectsRcd'),
+                                                tag = cms.string('BeamSpotOnlineTestHLT'),
+                                                refreshTime = cms.uint64(1)
+                                               )
 
+                                ),
+                                        connect = cms.string('oracle://cms_orcon_prod/CMS_CONDITIONS')
```

Comment:
this in general won't work unless you have the appropriate credentials. 
For a test can't you use `frontier` instead?

Reply:
checking now